tests/rbac/common/manager/rbac_manager_test.py

```diff
@@ -16,38 +16,36 @@
 import logging
 import pytest
 
-from rbac.common.manager.rbac_manager import RBACManager
+from rbac.common import rbac
 from rbac.common.user.user_manager import UserManager
 from rbac.common.role.role_manager import RoleManager
-from tests.rbac.common.manager.test_base import TestBase
+from tests.rbac.common.assertions import TestAssertions
 
 LOGGER = logging.getLogger(__name__)
 
 
 @pytest.mark.rbac_manager
-class RBACManagerTest(TestBase):
+class RBACManagerTest(TestAssertions):
     def __init__(self, *args, **kwargs):
-        TestBase.__init__(self, *args, **kwargs)
+        TestAssertions.__init__(self, *args, **kwargs)
 
-    @pytest.mark.unit
+    @pytest.mark.library
     def test_interface(self):
         """Verify the expected interface"""
-        self.assertIsInstance(self.rbac, RBACManager)
-
-        self.assertIsInstance(self.rbac.user, UserManager)
-        self.assertTrue(callable(self.rbac.user.address))
-        self.assertTrue(callable(self.rbac.user.make))
-        self.assertTrue(callable(self.rbac.user.make_addresses))
-        self.assertTrue(callable(self.rbac.user.make_payload))
-        self.assertTrue(callable(self.rbac.user.create))
-        self.assertTrue(callable(self.rbac.user.send))
-        self.assertTrue(callable(self.rbac.user.get))
-
-        self.assertIsInstance(self.rbac.role, RoleManager)
-        self.assertTrue(callable(self.rbac.role.address))
-        self.assertTrue(callable(self.rbac.role.make))
-        self.assertTrue(callable(self.rbac.role.make_addresses))
-        self.assertTrue(callable(self.rbac.role.make_payload))
-        self.assertTrue(callable(self.rbac.role.create))
-        self.assertTrue(callable(self.rbac.role.send))
-        self.assertTrue(callable(self.rbac.role.get))
+        self.assertIsInstance(rbac.user, UserManager)
```

Comment:
So this is a test that is testing python and how it works, more then the actual code.  It is not necessary.  This is more for your knowledge then a remove.